tsconfig.json

```diff
@@ -1,9 +1,9 @@
 {
   "extends": "./tsconfig.base.json",
   "compilerOptions": {
-    "rootDir": "./src",
-    "outDir": "./dist",
-    "stripInternal": true
+    "types": ["@types/node"],
```

Comment:
Why are these types explicitly referenced? They should be automatically loaded because they're global.